Burn Bryte by Roll20/BurnBryte.html

```diff
@@ -1,1120 +1,1101 @@
-                                <label data-i18n="reward">Reward</label><textarea name="attr_event_2_reward" placeholder="Event rewards"></textarea>
-                            </div>
-                            <hr />
-                            <div class="story-event">
-                                <label data-i18n="event-3">Event 3</label><input type="text" name="attr_event_3" placeholder="Event name" /><br />
-                                <label data-i18n="how-to-c">How to complete</label><textarea name="attr_event_3_howto" placeholder="How to complete the event"></textarea>
-                                <label data-i18n="reward">Reward</label><textarea name="attr_event_3_reward" placeholder="Event rewards"></textarea>
-                            </div>
-                            <hr />
-                            <div class="story-event">
-                                <label data-i18n="event-4">Event 4</label><input type="text" name="attr_event_4" placeholder="Event name" /><br />
-                                <label data-i18n="how-to-c">How to complete</label><textarea name="attr_event_4_howto" placeholder="How to complete the event"></textarea>
-                                <label data-i18n="reward">Reward</label><textarea name="attr_event_4_reward" placeholder="Event rewards"></textarea>
-                            </div>
-                            <hr />
-                            <div class="story-event">
-                                <label data-i18n="event-5">Event 5</label><input type="text" name="attr_event_5" placeholder="Event name" /><br />
-                                <label data-i18n="how-to-c">How to complete</label><textarea name="attr_event_5_howto" placeholder="How to complete the event"></textarea>
-                                <label data-i18n="reward">Reward</label><textarea name="attr_event_5_reward" placeholder="Event rewards"></textarea>
-                            </div>
-                        </div>
-                        <div class="display">
-                            <input class="display-flag" type="checkbox" name="attr_story_flag">
-                            <h4><span name="attr_story"></span></h4>
-                            <div class="details">
-                                <span name="attr_story_path"></span>
-                                <hr />
-                            </div>
-                            <div class="story-event">
-                                <input class="display-flag" type="checkbox" name="attr_display_flag_event_1">
-                                <label data-i18n="event-1">Event 1:</label><div class="story-act"><span name="attr_event_1"></span></div>
-                                <div class="square"><input type="checkbox" name="attr_event_1_checkbox" /><div class="box-check">3</div></div>
-                          
```

Comment:
Can you combine this function with the two below since they are doing the same thing?

You could do something like this

```
		['physical', 'mental', 'social'].forEach(attr => {
			on(`change:${attr}`, (eventinfo) => {
				const skillsPhyiscal = ["athletics", "melee", "power", "ranged", "skullduggery", "stealth"];
				const skillsMental = ["computers", "engineering", "medicine", "knowledge", "perception", "streetwise"];
				const skillsSocial = ["deceive", "decorum", "empathy", "perform", "presence", "suave"];
				const skills = (attr === 'physical') ? skillsPhyiscal : (attr === 'mental') ? skillsMental : skillsSocial

				const parseAttr = parseInt(eventinfo.newValue) || 0;
				const multDie = (social * 2);
				let newSkills = [];

				skills.forEach(skill => {
					newSkills[skill] = newDice;
				});

				setAttrs(newSkills);
			});
		});
```